printer_mods/sttts/Waveshare-5.5-inch-HDMI-AMOLED/README.md

```diff
@@ -0,0 +1,19 @@
+Waveshare 5.5inch HDMI AMOLED Mount
+===================================
+
+This replaces the default Klipper display with a [Waveshare 5.5inch HDMI AMOLED display](https://www.waveshare.com/wiki/5.5inch_HDMI_AMOLED) as a skirt to the Voron 2.4.
+
+As the cover around the display itself has pretty tight fit and ABS shrinks 1-2%, you might have to print it 1% or 2% bigger.
```

Comment:
All Voron parts and mods need to be designed so the STLs can be printed as is.  If printing the included STL has fitment issues, then please increase the size of the part appropriately so that it can be printed as is.

Reply:
done